icu4j/build.xml

```diff
@@ -1730,52 +1706,6 @@
         </java>
     </target>
 
-    <target name="gatherapiOld" depends="info, build-tools" description="Run API database generator tool (Pre Java 5 style)">
```

Comment:
This is for old API signature file format. I think we stopped to producing API signature file in old format since more than 10 versions ago. I think we don't need to maintain this target at this point (this means, you cannot compare APIs between ICU 4.4 and ICU 72.1 - but who would want this at this point?)